pkg/web/templates/partials/transaction/transaction_accept.html

```diff
@@ -1,8 +1,8 @@
 <section>
-  <section class="lg:flex lg:justify-between lg:items-center">
-    <div class="pb-4 flex items-center gap-x-1 lg:pb-0">
-      <dt class="font-bold text-lg">Transaction ID:</dt>
-      <dd>{{ .ID }}</dd>
+  <section class="p-4 border border-black bg-[#D9D9D9] lg:flex lg:justify-between lg:items-center">
```

Comment:
Should this section also have rounded borders like the section below?

Reply:
This style originally came from the preview modal and the border wasn't rounded there. I can update this to match the sections below.